code/game/objects/items/tanks/tank_types.dm

```diff
@@ -118,7 +118,7 @@
 	worn_icon = null
 	slot_flags = ITEM_SLOT_BELT
 	force = 5
-	volume = 6
+	volume = 24	//enough so they need to refill but not that often to be a chore
```

Comment:
You do know that plasmaman belt tanks can be used for breathing things other than plasma by other species, right?

Reply:
Don't burn the office.
Only cargo and engi remember this.

Reply:
Most people don't even fill their O2 tanks at all, if somebody really wants to go get a plasmaman tank and fill it with O2 and waste a far more valuable slot (the belt slot) then more power to them I guess?

Reply:
Do note that on live servers right now, players can get a plasmaman refill tank and use it to get similar behaviour described here. Tank sizes go 1L/2L/6L (plasma belt tank)/8L (double emergency oxygen tank)

After this change, they go 3L/6L/24L (plasma belt tank)/24L (double emergency oxygen tank)

If players wanted bigger internals, they can already do this on live.

Yes, this IS a very minor buff to people using plasma belt tanks as for their internals compared to before but, I think that is a price I'm willing to pay.